superbench/benchmarks/micro_benchmarks/cpu_stream_performance.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Shall we add this benchmark to document as well? (https://github.com/microsoft/superbenchmark/blob/main/docs/user-tutorial/benchmarks/micro-benchmarks.md)